generateTemplatesConfig.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+
+const generateTemplatesConfig = (ocpVersion, templateName, jenkinsEnabled, tektonEnabled, actionsEnabled, gitlabEnabled, githubEnabled, gitlabciEnabled) => {
+    const config = {
+        templates: [
+            "dotnet-basic",
+            "go",
+            "nodejs",
+            "python",
+            "java-quarkus",
+            "java-springboot"
+        ],
+        priority: [
+            "High"
+        ],
+        github: {
+            tekton: tektonEnabled === 'true',
+            jenkins: jenkinsEnabled === 'true',
+            actions: actionsEnabled === 'true',
+            host: "https://api.github.com"
+        },
+        gitlab: {
+            tekton: tektonEnabled === 'true',
+            jenkins: jenkinsEnabled === 'true',
+            gitlabci: gitlabciEnabled === 'true',
+            host: "https://gitlab.com",
+        },
+        pipeline: {
+            ocp: ocpVersion,
+            version: "1.0",
```

Comment:
Id it rhtap version?

Reply:
no idea, it should be rhtap version, right?

Reply:
yes that's what the idea was initially while poc.

Reply:
Done, update with latest rhtap version `1.4`